user_guide_src/source/testing/overview/018.php

```diff
@@ -2,26 +2,19 @@
 
 use CodeIgniter\CLI\CLI;
 use CodeIgniter\Test\CIUnitTestCase;
-use CodeIgniter\Test\Filters\CITestStreamFilter;
+use CodeIgniter\Test\StreamFilterTrait;
 
 final class SomeTest extends CIUnitTestCase
 {
-    protected function setUp(): void
-    {
-        CITestStreamFilter::$buffer = '';
-        $this->stream_filter        = stream_filter_append(STDOUT, 'CITestStreamFilter');
-    }
-
-    protected function tearDown(): void
-    {
-        stream_filter_remove($this->stream_filter);
-    }
+    use StreamFilterTrait;
 
     public function testSomeOutput(): void
     {
+        $this->resetStreamFilterBuffer();
```

Comment:
How about this?
```suggestion
        // Setup method of StreamFilterTrait clears the buffer.
        $this->assertSame('', $this->getStreamFilterBuffer());
```

Reply:
What do you mean?
Can you update the sample code to what you consider good?

Reply:
I think that the example should show not only getting data from the buffer, but also resetting it.

```php
public function testSomeOutput(): void
{
    CLI::write('first.');

    $expected = "\nfirst.\n";
    $this->assertSame($expected, $this->getStreamFilterBuffer());
    
    $this->resetStreamFilterBuffer();

    CLI::write('second.');
    
    $expected = "second.\n";
    $this->assertSame($expected, $this->getStreamFilterBuffer());
}

```

Reply:
Okay, I got it.

Can you update the sample code?
And let's finish this PR.